OMCompiler/Compiler/Script/CevalScriptBackend.mo

```diff
@@ -3805,17 +3805,17 @@ algorithm
     if isWindows then
       configureLogFile := "\""+configureLogFile+"\"";
     end if;
-    // do not create static.log in resource directory when --fmiFilter=blackBox
-    if intLt(Flags.getConfigEnum(Flags.FMI_FILTER), Flags.FMI_BLACKBOX) then
-      configureFMU(platform, fmutmp, configureLogFile, isWindows);
+    configureFMU(platform, fmutmp, configureLogFile, isWindows);
+    if Flags.getConfigEnum(Flags.FMI_FILTER) == Flags.FMI_BLACKBOX then
+      System.removeFile(configureLogFile);
```

Comment:
system.removeFile(configureLogFile) , does not remove the static.log file in windows,  why are we running the configureFMU() if the blackBox option is set,  it is just creating this static.log right ?

Reply:
```
 cmd := "rm " + configureLogFile;
 if 0 <> System.systemCall(cmd, outFile=logfile) then
    Error.addInternalError("Failed to remove Json file: " + fmutmp , sourceInfo());
 end if ;
```
However using the cmd "rm  " works

Reply:
@adrpo @sjoelund  why this "System.removeFile()" not working in mingw ?

Reply:
Damn, it includes the quotes, that's why it cannot find the file.